app/views/calls/_new_call.html_BASE_7141.erb

```diff
@@ -0,0 +1,16 @@
+<div class="modal fade new-call" tabindex="-1" role="dialog" id="call-<%= p.patient.primary_phone %>">
```

Comment:
Same deal for this -- not familiar with files that are named like this, so please remove (but let me know if this serves a purpose I'm not familiar with!